API/Svendeproeve-KlatreApp-API/Svendeproeve-KlatreApp-API/Controllers/WeatherForecastController.cs

```diff
@@ -9,13 +12,15 @@ public class WeatherForecastController : ControllerBase
         private static readonly string[] Summaries = new[]
         {
         "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
-    };
+        };
 
         private readonly ILogger<WeatherForecastController> _logger;
+        private readonly FirestoreService _fireStoreService;
 
-        public WeatherForecastController(ILogger<WeatherForecastController> logger)
+        public WeatherForecastController(ILogger<WeatherForecastController> logger, FirestoreService firestoreService)
```

Comment:
WTF IS THIS NAMING BRUH

Reply:
Jeg har fucking fjernet den fil alligevel